BitwardenShared/UI/Vault/Vault/AutofillList/VaultAutofillListProcessor.swift

```diff
@@ -152,6 +163,76 @@ class VaultAutofillListProcessor: StateProcessor<
         }
     }
 
+    @available(iOSApplicationExtension 17.0, *)
+    private func initFido2State() {
+        guard let fido2appExtensionDelegate = appExtensionDelegate as? Fido2AppExtensionDelegate else {
+            return
+        }
+
+        services.fido2UserInterfaceHelper.setupDelegate(fido2UserVerificationMediatorDelegate: self)
+
+        if let request = fido2appExtensionDelegate.getRequestForFido2Creation(),
+           let credentialIdentity = request.credentialIdentity as? ASPasskeyCredentialIdentity {
+            Task {
```

Comment:
🎨 Since this function is called from an effect, I would make this function async rather than creating a task here.

You could also call this function from the `loadData` effect rather than create a separate effect.

Reply:
Oh I didn't want to create a `Task` without the need to do so. Like if this view is not called from a Fido2 flow then it would be unnecessary to async/await it. But yeah we save too little in comparison to keep up with the pattern followed elsewhere. 
I started putting this in `loadData` but was unsure if that was a good spot to wrap `initFido2State` in a `Task`, given that I don't want/need to await `initFido2State` to finish; just a fire and forget approach.

Reply:
`Task`s are lightweight so I wouldn't be too concerned about creating one unnecessarily. And since the task is created by the view when you do `.task { await store.perform(.initFido2) }`, you're already in a task, so by making `initFido2State` not be async, you end up creating a task to perform the effect and then another one for this work when it could be done all in a single task.